src/entities/DFGVariable.jl

```diff
@@ -32,6 +32,7 @@ mutable struct VariableNodeData{T<:InferenceVariable}
     dontmargin::Bool
     solveInProgress::Int
     solvedCount::Int
+    solverKey::Symbol
```

Comment:
Changing type likely implies a new minor version series, shall we target DFG v0.9.0 for this then?

Reply:
I also just saw now that it is sometimes named `solverkey` and `solvekey`, 7 - 97

Reply:
@Affie Yeah that was a big headache, I changed the ones I saw to `solverKey`. 
@dehann Yep, let's aim for 0.9.0